samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java

```diff
@@ -1702,70 +1702,58 @@ private void getTFMCScores(Map publishedAnswerHash, List scores, HistogramQuesti
 							.toString((int) (((double) correctresponses / (double) qbean.getNumResponses()) * 100)));
 	}
 
+	private void getCalculatedQuestionScores(List<ItemGradingData> scores, HistogramQuestionScoresBean qbean, ItemDataIfc item) {
+		final String CORRECT = "Correct";
+		final String INCORRECT = "Incorrect";
+		final int COLUMN_MAX_HEIGHT = 100;
+
+		// count incorrect and correct to support column height calculation
+		Map<String, Integer> results = new HashMap<>();
+		results.put(CORRECT, Integer.valueOf(0));
+		results.put(INCORRECT, Integer.valueOf(0));
+
+		Map<Integer, String> answersMap = new HashMap<>();
+		int total = 0;
+		for (ItemGradingData score : scores) {
+			delegate.extractCalcQAnswersArray(answersMap, item, score.getAssessmentGradingId(), score.getAgentId());
+			if (score.getAutoScore() != null) {
+				total++;
+				if (delegate.getCalcQResult(score, item, answersMap, 1)) {
```

Comment:
I think this should be: 

{code}
if (delegate.getCalcQResult(score, item, answersMap, 1)) {
{code}